samples/client/petstore/typescript-fetch/builds/default/apis/PetApi.ts

```diff
@@ -23,7 +23,7 @@ import {
 } from '../models';
 
 export interface AddPetRequest {
-    body: Pet;
+    pet: Pet;
```

Comment:
please use the `bin/typescript-fetch-petstore-all.sh` command to generate the samples